service-front/module/Application/view/application/pages/national_insurance_number_success.twig

```diff
@@ -1,36 +1,48 @@
-            <svg class="moj-banner__icon" fill="currentColor" role="presentation" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25 25" height="25" width="25">
-                <path d="M25,6.2L8.7,23.2L0,14.1l4-4.2l4.7,4.9L21,2L25,6.2z" />
-            </svg>
-            <div class="moj-banner__message">
-                Initial identity confirmation complete
-            </div>
+    <div class="moj-banner moj-banner--success" role="region" aria-label="Success">
+        <svg class="moj-banner__icon" fill="currentColor" role="presentation" focusable="false"
+             xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25 25" height="25" width="25">
+            <path d="M25,6.2L8.7,23.2L0,14.1l4-4.2l4.7,4.9L21,2L25,6.2z"/>
+        </svg>
+        <div class="moj-banner__message">
+            Initial identity confirmation complete
         </div>
+    </div>
 
-        <p class="govuk-body">
-            Thank you - your national insurance number check is complete and verified. We now have four questions that we need to ask to
-            confirm your identity.
-        </p>
-
-        <p class="govuk-body">
-            I will read the questions to you and then read you a choice of answers. Take your time to answer them, if
-            you need to look at any documents to help you answer them that is fine.
-        </p>
-
-        <p class="govuk-body">
-            I can read you a choice of answers and you can pick your answer, or if you'd prefer you can just tell me the
-            answer after the question.
-        </p>
-        <p class="govuk-body">
-            <a href="./id-verify-questions" class="govuk-button">Continue</a>
-
-            <a href="./donor-id-check" role="button" draggable="false"
-               class="govuk-button moj-button-menu__item govuk-button--secondary " data-module="govuk-button">
-                Return To Start
-            </a>
-        </p>
+    <p class="govuk-body">
+        Thank you - your initial identity check is complete and your identity verified.
+    </p>
+
+    <p class="govuk-body">
+        We now have four questions that we need to ask to confirm your identity.
+    </p>
+
+    <p class="govuk-body">
+        I will read the question to you and then read you a choice of answers. Take your time to answer them,
+        if you need to look at any documents to help you answer the question that is fine.
+    </p>
+
+    <p class="govuk-body">
+        I can read you a choice of answers and you can pick your answer,
+        or if you'd prefer you can just tell me the answer after the question.
+    </p>
+    <p class="govuk-body">
+        <a href="./id-verify-questions" class="govuk-button">Continue</a>
+
+        <a href="./donor-id-check" role="button" draggable="false"
+           class="govuk-button moj-button-menu__item govuk-button--secondary " data-module="govuk-button">
+            Return To Start
+        </a>
+    </p>
 
 </div>
+
+
+
+
+
+
+
+
```

Comment:
You might be missing a setting in your editor/IDE: it should automatically remove excess newlines at the end of a file